controllers/toolchaincluster/toolchaincluster_controller.go

```diff
@@ -16,6 +17,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+// TODO: move this to somewhere more appropriate - maybe somewhere in the api module?
+// We can live with this here for now, because this label is not used for anything yet.
+var ToolchainClusterLabel = toolchainv1alpha1.LabelKeyPrefix + "toolchain-cluster"
```

Comment:
Yeah, let's move it to the api repo https://github.com/codeready-toolchain/api/blob/master/api/v1alpha1/toolchaincluster_types.go